app/grandchallenge/algorithms/templates/algorithms/data_tables/result_list.html

```diff
@@ -0,0 +1,90 @@
+{% load url %}
+{% load profiles %}
+{% load workstations %}
+{% load guardian_tags %}
+{% load pathlib %}
+{% load humanize %}
+{% load evaluation_extras %}
+
+{{ result.job.created|naturaltime }}
```

Comment:
Maybe this is answered later but how does ordering work on these data cells now? Aren't things like `data-order="{{ result.job.created|date:"U" }}` lost now?